pkg/featuresupport/featuresupport.go

```diff
@@ -0,0 +1,53 @@
+package featuresupport
```

Comment:
I created this package to easily provide different features for different providers and various release versions.